public_html/wp-content/mu-plugins/wcorg-misc.php

```diff
@@ -4,6 +4,22 @@
  * Miscellaneous snippets that don't warrant their own file
  */
 
+/**
+ * Get the current environment.
+ *
+ * Defaults to 'development' if the `WORDCAMP_ENVIRONMENT` constant isn't set or is empty.
+ *
+ * @return string
+ */
+function get_wordcamp_environment() {
+	$environment = 'development';
+
+	if ( defined( 'WORDCAMP_ENVIRONMENT' ) && WORDCAMP_ENVIRONMENT ) {
+		$environment = WORDCAMP_ENVIRONMENT;
+	}
+
+	return $environment;
+}
```

Comment:
> What's the reason for falling back to dev here instead of local ? Same for set_client_config().

I was thinking that `development` is the catchall non-production value, whereas `local` has specific implications since the domain is different, no 2-way communication, etc.

> 3-helpers-misc.php seems like a more consistent location for this type of thing.

Oh, good call. It's kind of confusing that we have **3-helpers-misc.php** _and_ **wcorg-misc.php**. But that's a conversation for another day 😄

Reply:
1228241190643ce36608c8fee54750db47b68cca

Reply:
Yeah, I can see that. I don't think the two names were intended to imply a relationship between the two files, just to distinguish miscellaneous helpers from `wcpt`-specific helpers.

Feel free to rename it to whatever makes more sense to you.

Reply:
> I was thinking that `development` is the catchall non-production value, whereas `local` has specific implications since the domain is different, no 2-way communication, etc.

That feels counter-intuitive and unconventional to me. If you'd prefer that, I think all the possible values of `WORDCAMP_ENVIRONMENT` should be documented alongside the `wp-config` definition.

Reply:
Documented: 6f7dccda